engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorImpl.java

```diff
@@ -530,11 +532,19 @@ private boolean validateConstraint(ValidationContext<?> validationContext,
 				ReflectionHelper.getPropertyName( metaConstraint.getLocation().getMember() )
 			);
 
+			// The user didn't set un
```

Comment:
Right, while I was writing this comment I had some doubts about the overall behaviour and I've sent the pull request forgetting to remove it.

I will add commits to clean the various left-overs